python/ray/import_thread.py

```diff
@@ -157,13 +157,13 @@ def f():
 
     def fetch_and_execute_function_to_run(self, key):
         """Run on arbitrary function on the worker."""
-        driver_id, serialized_function = self.redis_client.hmget(
-            key, ["driver_id", "function"])
-
-        if (self.worker.mode in [ray.SCRIPT_MODE, ray.SILENT_MODE]
+        driver_id, serialized_function, include_driver = self.redis_client.hmget(
+            key, ["driver_id", "function", "include_driver"])
```

Comment:
Let's convert `include_driver` to a bool before using it in the if statement below. Also note that there may be some Python 2/3 differences.